### PR TITLE
Remove update and forgot password APIs

### PR DESCRIPTION
**Pull request by idanovo**
Closes #[6532](https://github.com/treeverse/lakeFS/issues/6532)

### CODE REVIEW

File: api/swagger.yml [hunk 1]
```diff
@@ -691,15 +691,13 @@ components:
       properties:
         id:
           type: string
-          description: a unique identifier for the user. In password-based authentication, this is the email.
+          description: a unique identifier for the user.
```
Comment (arielshaqed) [inline api/swagger.yml:694]: Nice catch! ✨

**Review by arielshaqed — APPROVED** [review]
Neat, thanks!

* login.tsx needs some care, GitHub Actions / Analyze says maybe the `getAuthCapabilities()` call is unused.

**Review by arielshaqed — CHANGES_REQUESTED** [review]
Forgot, so blocking for now

**ALSO** please go over webui/ and look for these URLs: webui doesn't use a generated client but _did_ use these URLs, so needs to be changed manually and the TypeScript compiler cannot find the change from the OpenAPI spec change.

**Comment by idanovo** [comment]
> Forgot, so blocking for now
> 
> **ALSO** please go over webui/ and look for these URLs: webui doesn't use a generated client but _did_ use these URLs, so needs to be changed manually and the TypeScript compiler cannot find the change from the OpenAPI spec change.

@arielshaqed Not sure I understand this comment. I removed the calls for update/forgot password from the webui code.
Am I missing anything?

**Comment by arielshaqed** [comment]
> > Forgot, so blocking for now
> > **ALSO** please go over webui/ and look for these URLs: webui doesn't use a generated client but _did_ use these URLs, so needs to be changed manually and the TypeScript compiler cannot find the change from the OpenAPI spec change.
> 
> @arielshaqed Not sure I understand this comment. I removed the calls for update/forgot password from the webui code. Am I missing anything?

Nope, you're right and I'm wrong.

THANKS!

**Review by arielshaqed — APPROVED** [review]
Love the diff stats!
GitHub shows 22 lines "added", 2454 **removed**

**Pull request merged**